IndyAV/control/indyav_control/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ add_message_files(
   DIRECTORY msg
   FILES
   SteeringStamped.msg
-  ThrustStamped.msg
+  TorqueStamped.msg
```

Comment:
With this message, we are commanding a rotational velocity (not a rotational force) in some units (I would suspect rads/sec) the message name should reflect that